src/main/java/co/cask/hydrator/OrientDBConfig.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.hydrator;
+
+import co.cask.cdap.api.annotation.Description;
+import co.cask.cdap.api.annotation.Name;
+import co.cask.cdap.api.plugin.PluginConfig;
+
+/**
+* {@link PluginConfig} for {@link OrientDBSink}.
+*/
+public class OrientDBConfig extends PluginConfig {
+  public static final String REFERENCE_NAME = "referenceName";
+  public static final String CONNECTION_STRING = "connectionString";
+  public static final String USERNAME = "username";
+  public static final String PASSWORD = "password";
+  public static final String VERTEX = "vertex";
+  public static final String EDGE = "edge";
+
+  @Name(REFERENCE_NAME)
+  @Description("Reference Name for the OrientDB Sink")
+  private String referenceName;
+
+  @Name(CONNECTION_STRING)
+  @Description("Specific the OrientDB connection string")
```

Comment:
What's the sample connection string. Also can we have the sample prefilled in the text box. 
 Example:  " --Sample connection string 10102:3333"
Specify the default port.